packages/web-react/src/components/Breadcrumbs/BreadcrumbsItem.tsx

```diff
@@ -0,0 +1,52 @@
+import classNames from 'classnames';
+import React from 'react';
+import { useClassNamePrefix } from '../../hooks';
+import { useStyleProps } from '../../hooks/styleProps';
+import { SpiritBreadcrumbsItemProps } from '../../types';
+import { Icon } from '../Icon';
+import { Link } from '../Link';
+
+const defaultProps = {
+  iconStart: undefined,
+  iconEnd: <Icon name="chevron-right" />,
+  isCurrent: false,
+  isGoBackOnly: false,
+};
+
+const BreadcrumbsItem = (props: SpiritBreadcrumbsItemProps) => {
+  const { children, href, isCurrent, iconStart, iconEnd, isGoBackOnly, ...restProps } = props;
+  const { styleProps, props: otherProps } = useStyleProps(restProps);
+  const displayNoneClassName = useClassNamePrefix('d-none');
```

Comment:
no hook for these? Other components all use the same component hook.

Reply:
Please be more specific. I do not understand what the problem is. There is no hook for utilities.

Reply:
Ok, I understand. Refactored.